Update Daimakaimura -Japan Resale Ver.-.mra

jepapac opened the pull request:
Added missing dip switch settings including the undocumented Stage Select settings. Reorganized the order to match the arcade manual.

jepapac committed:
Update Daimakaimura -Japan Resale Ver.-.mra

Added missing dip switch settings including the undocumented Stage Select settings. Reorganized the order to match the arcade manual.

jotego commented:
Hi, thanks for the commit
It looks like there is an XML error as you are missing the inverted commas around the `values` attribute in a line or two.

Speaking of `values`, I don't think this attribute is supported by the [MRA tool](https://github.com/sebdel/mra-tools). This tools is used to generate a binary file on FPGA platforms that do not parse MRA files directly (basically all except MiSTer).

I do not think I can take up the PR because the `values` parameter would break the file for other platforms. The MRA tool is not maintained any more, so until a new project comes to replace it, I think there will be no `values` support for non MiSTer platforms.

jepapac commented:
Thank you for the explanation. I'll see if I can make a better update that
will work with the MRA tool. I wasn't aware of it. I found that
documentation is pretty lackluster for the mra files.

On Wed, Jun 15, 2022 at 11:08 AM Jose Tejada ***@***.***>
wrote:

> Hi, thanks for the commit
> It looks like there is an XML error as you are missing the inverted commas
> around the values attribute in a line or two.
>
> Speaking of values, I don't think this attribute is supported by the MRA
> tool <https://github.com/sebdel/mra-tools>. This tools is used to
> generate a binary file on FPGA platforms that do not parse MRA files
> directly (basically all except MiSTer).
>
> I do not think I can take up the PR because the values parameter would
> break the file for other platforms. The MRA tool is not maintained any
> more, so until a new project comes to replace it, I think there will be no
> values support for non MiSTer platforms.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/jotego/jtcps/pull/135#issuecomment-1156778518>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AXSWYYU3VDOOAPQ3KTE244DVPIL2HANCNFSM5Y4FEUSA>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>

jepapac committed:
Update Daimakaimura -Japan Resale Ver.-.mra

Added additional dip switch settings including the undocumented Stage Select without the use of "values" parameter to be consistent with the use of the MRA tool.

jepapac commented:
Updated the mra to remove the "values" parameter and fixed the formatting.

jotego commented:
Closing this as I am consolidating this repo into JTGNG
